src/graphic-device/dot-use-examples/index.md

```diff
@@ -86,33 +90,34 @@ Get help with...
 
 ![TODO](./video-thumb-get-help.png)
 
-60s guides
+60 second guides
 
 ![TODO](./video-thumb-60s.png)
 
 Step-by-step guides
 
 ![TODO](./video-thumb-step.png)
 
-Influencer/presenter
+Influencer or presenter
 
 ![TODO](./video-thumb-influencer.png)
 
-Indicative examples for illustrative purposes only.
-
 ## Static dot examples
 
 The dot can also be used as a static graphic, adding emphasis and helping guide users through content.
 
+See these indicative examples of static dot graphics. This example is shown for illustrative purposes only.
```

Comment:
Same here, mix of plural "these examples" and singular "this example".

Reply:
```suggestion
See these indicative examples of static dot graphics. These examples are shown for illustrative purposes only.
```